docs/reports.md

```diff
@@ -376,3 +376,129 @@ npx codeceptjs run --reporter mocha-multi
 ```
 
 This will give you cli with steps in console and HTML report in `output` directory.
+
+
+## Testrail
+
+Testrail integration with CodeceptJS is never simple like this. The test run is created automatically after the execution. The screenshots of failed tests are also attached to test results.
```

Comment:
Please dont use "simple" in documention. Especially with "never" 😁 try to use more neutral style

Reply:
thanks @DavertMik . I fixed this issue. But out of curiosity, why can't we use 'simple' in documentation? any context behind that?

Reply:
I can't find the exact quote at this moment but I read a blogpost which advocated the apprach to not using words like "simple" and "easy"

1. in current context it means nothing - if it is simple the user will figure this out by following docs
2. it can't be as simple to someone as you think :smile:

Reply:
> I can't find the exact quote at this moment but I read a blogpost which advocated the apprach to not using words like "simple" and "easy"
> 
> 
> 
> 1. in current context it means nothing - if it is simple the user will figure this out by following docs
> 
> 2. it can't be as simple to someone as you think :smile: 

Aha, I found this https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher. I guess it's the thing you mentioned.

Reply:
Thank you! The right one